pkgs/by-name/li/limine/package.nix

```
@@ -47,7 +47,8 @@ let
     ++ lib.optionals pxeSupport [ "--enable-bios-pxe" ]
     ++ lib.concatMap uefiFlags (
       if targets == [ ] then [ stdenv.hostPlatform.parsed.cpu.name ] else targets
-    );
+    )
+    ++ [ "CC_FOR_TARGET=${llvmPackages.clang-unwrapped}" ];
```

```suggestion
    ++ [ "TOOLCHAIN_FOR_TARGET=${stdenv.cc.cc}" ];
```
However, it seems [freestanding-toolchain](https://github.com/osdev0/freestanding-toolchain) does not like this